<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stacked">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main_page_ui">
     <widget class="QPushButton" name="find_server_btn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти сервер по хостнейму</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_title">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Крестики нолики</string>
      </property>
     </widget>
     <widget class="QPushButton" name="create_server_btn">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Создать сервер</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="port_input">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostname_input">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_port">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_host">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>240</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>hostname</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="button_page_ui">
     <widget class="QPushButton" name="mainbutton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">    QPushButton {
        background-color: #8B0000;
		border-radius: 100px; /* Делаем круглую форму */
       	padding: 0px;
        color: white;
        border: none;
        padding: 15px 32px;
        text-align: center;
        text-decoration: none;
        display: inline-block;
        font-size: 16px;
        cursor: pointer;
    }
    QPushButton:hover {
        background-color: #B22222;
    }
    QPushButton:pressed {
        background-color: #b30000;
        padding-top: 17px;
        padding-bottom: 13px;
    }
    

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="png_page_ui">
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>611</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>hostname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>410</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>322</x>
        <y>405</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>410</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
